frontend/src/store/modules/app.js

```diff
@@ -35,16 +32,27 @@ export default {
       if (!state.alertNotification) {
         state.alertNotification = true;
       }
+    },
+    setProgramYearList: (state, programYearList) => {
+      state.programYearList = programYearList;
+    },
+    setChildCareCategoryList: (state, childCareCategoryList) => {
+      state.childCareCategoryList = childCareCategoryList;
+    },
+    setOrganizationTypeList: (state, organizationTypeList) => {
+      state.organizationTypeList = organizationTypeList;
     }
+
   },
   actions: {
-    async getMincodeSchoolNames({ commit, state}) {
+    async getLookupInfo({commit}){
```

Comment:
this function is not called anywhere?

Reply:
This is called via the mapGetters, on line 126 of OrganizationInformation.vue
  ...mapGetters('auth', ['userInfo']),